logger/src/lib.rs

```diff
@@ -136,17 +136,16 @@
 //! metric will get increased.
 //! Metrics are only logged to pipes. Logs can be flushed either to stdout/stderr or to a pipe.
 
-extern crate chrono;
 // workaround to macro_reexport
 #[macro_use]
 extern crate lazy_static;
+extern crate fc_util;
```

Comment:
Please move this `extern crate` after all the crate imports that come from crates.io.